### ISSUE TITLE
map gender to sex, not sex role

### ISSUE THREAD

**Issue by VladimirAlexiev** [american-art/aac_mappings#50]
[Gender is mapped](http://review.americanartcollaborative.org/entity/E39_Actor#field_12-search_0) to [300055147](http://www.getty.edu/vow/AATHierarchy?find=300055146&logic=AND&note=&page=1&subjectid=300055147) sex role. However, that concept has no children and is a bit unclear.
In https://jira.getty.edu/browse/ITSLOD-497 I suggest to delete it, since there's already:
- [300055146](http://www.getty.edu/vow/AATHierarchy?find=300055146&logic=AND&note=&page=1&subjectid=300055146) sex, with children male, female
- [300400503](http://www.getty.edu/vow/AATHierarchy?find=300055146&logic=AND&note=&page=1&subjectid=300400503) sexual orientation, with children: LGBT, bisexual, heterosexual etc

I think AAC should map to 300055146 sex

**Comment by workergnome** [comment]
There's a long discussion with @cbutcosk about this at https://github.com/american-art/aac_mappings/issues/47.  

Basically, "sex" is about biology, and unless the museum is determining gender using DNA tests, it's not really correct to use it.  

"Sexual orientation" is about whom they wanted to sleep with, and that's also not what museums describe.

"sex role" is about the gender that they appeared/performed, which is the most relevant characteristic for museums.

**Comment by azaroth42** [comment]
Either way it would be a discussion for the future. Can we tag as 1.1?

**Comment by workergnome** [comment]
Yup.  Done.

**Comment by VladimirAlexiev** [comment]
300055147 doesn't have any child concepts. (Patricia answered: "sex role" does not require children – it is a concept useful on its own.)
So how are you sure it would be correct to map eg the value "male" to a property typed with 300055147?

**Comment by VladimirAlexiev** [comment]
@workergnome Please reopen for "version 2'.

> unless the museum is determining gender using DNA tests

This comment is disingenuous. I know you're a man without testing your DNA.
Since 300055147 doesn't have any children, why do you think it's correct to use it for male/female or man/woman?

**Comment by workergnome** [comment]
I disagree with you, but I also agree that we should keep open for discussion at v.2.

**Comment by VladimirAlexiev** [comment]
According to [the mapping](http://review.americanartcollaborative.org/entity/E39_Actor#field_12-search_0), it should be like this (eg from https://github.com/american-art/GM/issues/18):
```ttl
<constituent/whoever> crm:P2_has_type <constituent/whoever/gender>.
<constituent/whoever/gender> a crm:E55_Type;
  crm:P2_has_type <thesaurus/gender>.
<thesaurus/gender> a crm:E55_Type;
  skos:broadMatch aat:300055147; # sex role
  skos:prefLabel "gender".
```
Allowing museums to have their own notions of what is Gender strikes me as a crazy idea.

**Comment by workergnome** [comment]
At least here in the US, prescriptive requirements for gender are a major political issue—see #47, as well as https://medium.com/@malpinder/falsehoods-programmers-believe-about-gender-cf1a55085ab2 for an example of the sort of critique strictly defined gender roles will engender.

**Comment by VladimirAlexiev** [comment]
Allowing museums to use any set of Gender values is fine: they could use one of the 15ish in Wikidata if they want, see http://vladimiralexiev.github.io/pres/20150212-sex-or-gender/#sec-2-5.

Allowing them to have their own *notions* just doesn't match any current business requirement. Pray tell, what do they have in their data? 
You mean, like, fa'afafine (Q1399232), māhū (Q3277905), kathoey (Q746411), fakaleiti (Q350374), hijra (Q660882) etc etc is just not enough?
